### DISCUSSION TITLE
Hello from Sweden!

### DISCUSSION THREAD

**Discussion by helenabarmer** [OpenMined/courses#46] [Introduce Yourself]
Hi everyone!

So excited to be here and share this learning journey together with all of you ❤️  

My name is Helena and I am junior data scientist from Sweden (recently graduated!).
At the moment I am involved in the **Online Events Team** at OpenMined so I hope to see you in one of our online events! 
I am also a member of the **RecSys Team** at OpenMined led by the awesome Maddie Shang 😎 

When I am not coding or learning about privacy preserving technology I like to skateboard and hang out with my kitten and family 🛹 🐈 👪 

Looking forward to have fun and learn together! ❤️

**Comment by yemikifouly** [comment]
Hi Helena! Welcome to the discussion board and thank you for the awesome Welcoming Event!  🙂🙌

**Comment by iamtrask** [comment]
Go Helena!!! Fantastic job on the welcoming event!
